apps/condo/domains/property/components/PropertyAddressSearchInput.tsx

```diff
@@ -72,20 +72,10 @@ export const PropertyAddressSearchInput: React.FC<IAddressSearchInput> = (props)
                         searchValue === dataItem.text
                             ? dataItem.text
                             : (
-                                <Highlighter
+                                <TextHighlighter
```

Comment:
Maybe remove this line above?

`TODO(DOMA-1513) replace HighLighter with apps/condo/domains/common/components/TextHighlighter.tsx and renderHighlightedPart`